src/commands/serve.rs

```diff
@@ -76,30 +77,53 @@ pub(crate) fn serve(args: &ArgMatches) {
 }
 
 fn serve_foreground(args: &ArgMatches) {
-    info!("Starting PHP...");
+    let port = find_available_port(parse_default_port(args.value_of("port").unwrap_or(DEFAULT_PORT), DEFAULT_PORT));
+
+    let mut document_root = get_document_root(args.value_of("document-root").unwrap_or("").to_string());
+    if document_root.ends_with('/') { document_root.pop(); }
+    if document_root.ends_with('\\') { document_root.pop(); }
+    document_root.push_str(if cfg!(target_family = "windows") { "\\" } else { "/" });
+    let doc_root_path = PathBuf::from(document_root.as_str());
+    let scripts = vec!["index.php", "app_dev.php", "app.php"];
```

Comment:
Can you rename this variable to `common_scripts_names` instead, and put every element on its own line?

I'd like to keep this kind of "hardcoded" stuff really visible in case of contributions, or in case other PHP frameworks might want to have some "highlights". It would be like a _"We support {this framework}"_ flag :)

Reply:
Also, I really hesitate on putting `app_dev` before `app`.

If we want this app to be "prod-ready" one day, we shouldn't do this

Reply:
> Also, I really hesitate on putting `app_dev` before `app`.
> 
> If we want this app to be "prod-ready" one day, we shouldn't do this

twice solution for prod environment:

* remove `app_dev.php` file (good practice)
* use the `passtru` argument to provide the PHP entrypoint.

The last point can be add in Rymfony documentation for use in production environment.

Reply:
I think we could add a warning at Rymfony's startup, if user specified no entrypoint and `app_dev` is resolved, add a big warning explaining that `app_dev` is for dev only, and if they're using Rymfony on prod, they should delete it.

If the user specifies the entrypoint themselves, no warning (we consider they know what they're doing).

Like this:
![image](https://user-images.githubusercontent.com/3369266/105147291-48d5d400-5b01-11eb-8ae9-be2e48088399.png)

Still wondering about warning vs error, but you get the idea 😉

Reply:
I have added this message with warning level. I use the level error only when the process cannot continue normaly.

Using `app_dev.php` allow Rymfony to continue normaly. :wink: